_posts/blog/2018-19-10-Reseña-Digital.md

```diff
@@ -0,0 +1,31 @@
+Reseña del archivo digital "Archivo Mapuche Warria Mew. Waria ñi tukulpan dungu/ Ampliando los horizontes de la memoria."
+
+_Archivo Mapuche Warria Mew. Waria ñi tukulpan dungu/ Ampliando los horizontes de la memoria._ Proyecto Financiado por el Fondo Nacional de Desarrollo Cultural y las Artes.    Web. 06 de octubre de 2018 (http://www.mapuchewariamew.cl)
+
+Los investigadores del Pueblo Mapuche en Chile, Claudio Alvarado Lincopi y  Enrique Antileo Baeza, integrantes del Centro de Estudios e Investigación Comunidad de Historia Mapuche (CHM) gestionaron el proyecto denominado _“Santiago Waria Mew. Memoria y fotografía de la migración mapuche”._ El proyecto fue financiado con el apoyo del Fondo Nacional de Desarrollo Cultural y las Artes (FONDART), Línea Desarrollo de las Culturas Indígenas, el cual consiste en un programa del Estado para apoyar la promoción del desarrollo cultural, artistísco y patrimonial en Chile. El proyecto se constituye del libro _“Santiago Waria Mew. Memoria y fotografía de la migración mapuche”_ editados en los años 2016 y 2017 (son dos ediciones) respectivamente, y también del archivo digital http://www.mapuchewariamew.cl. De acuerdo a los autores Antileo & Alvarado (2017) el proyecto tiene como objetivo “_narrar y reflexionar sobre la migración mapuche desde los diversos archivos que nos han legado las familias mapuche: fotografías, cartas, diarios, boletines, dibujos, etc.”_. 
```

Comment:
En vez de incluir el URL en el texto, se puede usar markdown para crear un link. Instrucciones: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

Typos:
* artístico
* editado